docker/nginx/nginx-x509.conf

```diff
@@ -69,154 +55,53 @@ http {
 
         # Proxy pass to users service
         location ~ ^/(users|tokens) {
-            proxy_redirect off;
-            proxy_set_header Host $host;
-            proxy_set_header X-Real-IP $remote_addr;
-            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-            proxy_set_header X-Forwarded-Proto $scheme;
-            proxy_pass http://users:8180;
-
-            # Allow OPTIONS method CORS
-            if ($request_method = OPTIONS ) {
-                add_header Content-Length 0;
-                add_header Content-Type text/plain;
-                return 200;
-            }
+            include snippets/proxy-headers.conf;
+            proxy_pass http://users:${MF_USERS_HTTP_PORT};
         }
 
         # Proxy pass to things service
         location ~ ^/(things|channels) {
-            proxy_redirect off;
-            proxy_set_header Host $host;
-            proxy_set_header X-Real-IP $remote_addr;
-            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-            proxy_set_header X-Forwarded-Proto $scheme;
+            include snippets/proxy-headers.conf;
```

Comment:
Do we need ` include snippets/verify-ssl-client.conf;` here also ?

Reply:
Let me check, if there was one before then we do.

Reply:
Yes, I checked it out - no. SSL verification is used for mTLS for messaging. `/things` and `/channels` are still accessed by user with his JWT.